requirements.txt

```diff
@@ -0,0 +1,4 @@
+aiohttp==1.0.5
+pytest==3.0.4
+requests==2.12.1
+responses==0.5.1
```

Comment:
The question of pinning or no pinning is a big debate, but i'd recommend creating a requirements.in file, specifying un-pinned dependencies, and one requirements.txt with the pinned ones. That way anyone can choose, and you can guarantee a version that works and has been tested.